commons/ava_testsuite/byzantine_gecko_network_tests.go

```diff
@@ -0,0 +1,115 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_services"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	BYZANTINE_CONFIG_ID = 2
```

Comment:
Let's lowercase these to make them as private as possible since they should only be local to the test (why, oh why, did they use naming as a way to declare private/public :| ).

Actually, can you just push this entire file into its own package, so that way it's truly private?